docs_src/examples/Ch-ExamplesUsbDeviceService.md

```diff
@@ -0,0 +1,305 @@
+# USB Camera Device Service RTSP Streaming Guide
+## Overview
+The EdgeX usb device service is designed for communicating with USB cameras attached to Linux OS platforms. This guide will help configure and build the usb device service and start streaming video from the USB camera.
+
+This service provides the following capabilities:
+- V4L2 API to get camera metadata
+- Camera status
+- Video stream reference
+- FFmpeg framework to capture video frames and stream them to an RTSP server
+- An embedded [RTSP server](https://github.com/aler9/rtsp-simple-server)
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
+
+**Other Requirements**
+
+You must have administrator (sudo) privileges to execute the user guide commands.
+
+## Tested Devices
+The following devices have been tested with EdgeX USB Camera Device Service:  
+Note: Results may vary based on camera hardware/firmware version and operating system support.
+<!-- sorted alphabetically -->
+- AUKEY PC-LM1E Webcam
+- HP w200 Webcam
+- Jinpei JW-01B USB FHD Web Computer Camera
+- Logitech Brio 4K
+- Logitech C270 HD Webcam
+- Logitech StreamCam
+
+## Dependencies
+The software has dependencies, including Git, Docker, Docker Compose, and assorted command line tools. Follow the instructions linked here to install any dependency that is not already installed [see here](https://github.com/edgexfoundry/device-usb-camera/blob/main/docs/setup.md).
+
+### Install additional Tools
+Install the media utility tool:
+
+   ```bash
+   sudo apt install mplayer v4l-utils
+   ```
+
+- `mplayer` is used to view the video stream
+- `v4l-utils` is used to determine the video stream path of a usb camera
+
+## Get the Source Code
```

Comment:
This make me think we are going to get the source for device-usb-camera, but never do.
```suggestion
## Run EdgeX with USB Camera Device Service
```